js/chatbot/shared/ChatBot.svelte

```diff
@@ -195,6 +195,23 @@
 			liked: selected === "like"
 		});
 	}
+
+	function get_message_label_data(message: NormalisedMessage): string {
+		if (message.type === "text") {
+			return message.value;
+		} else if (message.type === "component") {
+			return `a component of type ${message.component}`;
+		} else if (message.type === "file") {
+			if (Array.isArray(message.file)) {
+				return `file of extension type: ${message.file[0].orig_name?.split(".").pop()}`;
+			}
+			return (
+				`extension: ${message.file?.orig_name?.split(".").pop()}` +
```

Comment:
Should this also `file of extension type: ... with name .... `? Just so that it's more consistent with the array case.